aleph_alpha_client/embedding.py

```diff
@@ -89,6 +88,21 @@ class SemanticRepresentation(Enum):
     Query = "query"
 
 
+class Compression(Enum):
```

Comment:
Honestly a bit too much. I am fine having the `128` not reflected in the type system. Just take an Integer. The docstring though should be moved to the parameter destcription.